src/pages/HomePage.js

```diff
@@ -39,9 +37,8 @@ function HomePage() {
             //const json = await result.json();
 
             console.log(result.data);
-           // window.open((result.data)['tracks']['items'][0]['external_urls']['spotify']);
-            //window.location.replace("/SongPage/" + (result.data)['tracks']['items'][0]['external_urls']['spotify']);
-            setTitle((result.data)['tracks']['items'][0]['name']);
+            window.open((result.data)['tracks']['items'][0]['external_urls']['spotify']);
+
```

Comment:
What exactly do these lines do? I haven't used window.open very much

Reply:
window.open() simply opens a new page in the same window. This was meant to check that we were able to retrieve the correct data from the search request to the spotify api, and while we don't have a fully functioning song page, I thought it would be helpful for us to have some page to redirect as a temporary place holder.